CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ FEATURES:
 IMPROVEMENTS:
 * CLI
   * Update minimum go version for project to 1.19 [[GH-1633](https://github.com/hashicorp/consul-k8s/pull/1633)]
+  * `consul-k8s uninstall` now deletes custom resources when uninstalling Consul. [[GH-1623](https://github.com/hashicorp/consul-k8s/pull/1623)] 
```

Comment:
[nit] I think we have a style guide on this someone where, but I think these things are supposed to start with `Enable .....` or `Add support ....`.  I would take a look at our other entries and possibly revise this to feel like the others.

Reply:
Changed to "Enable `consul-k8s uninstall` to delete custom resources when uninstalling Consul. This is done by default. [[GH-1623](https://github.com/hashicorp/consul-k8s/pull/1623)]"